bin/capsule.js

```diff
@@ -1098,12 +1173,18 @@ const ciCmds = async(type) => {
      printErrorAndDie('Project name is required!', true);
   }
 
-
   if (commander.type === 'init') {
      let initType = 'create'
      await s3Cmds(initType)
-     await ciCmds(initType)
      await webCmds(initType)
+     await ciCmds(initType)
+  }
+
+  if (commander.type === 'remove') {
+     let deleteType = 'delete'
+     await ciCmds(deleteType)
```

Comment:
just like in the lines above, these async commands will be executed in series (one after another).
If you're not waiting for one to finish before another is executed, I'd consider letting them all fire in parallel

```js
await Promise.all([
  ciCmds(deleteType),
  webCmds(deleteType),
  s3Cmds(deleteType)
])
```